Engines/Wine/QuickScript/Steam Script/script.js

```diff
@@ -129,6 +129,8 @@ module.default = class SteamScript extends QuickScript {
         const wine = new Wine()
             .wizard(setupWizard)
             .prefix(this._name, this._wineDistribution, this._wineArchitecture, this._wineVersion);
+        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/manual/PhoenicisOrg/scripts/pullRequest?prid=4816948)